src/lib/Scenes/Articles/ArticlesList.tsx

```diff
@@ -111,6 +113,38 @@ export const useNumColumns = () => {
   return orientation === "portrait" ? 2 : 3
 }
 
+export const ArticlesPlaceholder = () => {
+  const numColumns = 1
+
```

Comment:
```suggestion

// non hooks version of useNumColumns
const nonHookNumOfColumns = (): number => {
  const { width, height } = Dimensions.get("window")
  const isPortrait = height > width
  const isTablet = isPad()
  if ((isTablet && isPortrait) || (!isTablet && !isPortrait)) {
    return 2
  }
  if (isTablet && !isPortrait) {
    return 3
  }
  return 1
}

export const ArticlesPlaceholder = () => {
  const numColumns = nonHookNumOfColumns()

```

@rajsam003 in addition to @olerichter00 concerns about the ArticlesList placeholder, I have also noticed that it does not look good on tablets in both portrait and landscape mode. The placeholders do not match the list eventually displayed, because numColumns cannot be 1 on tablets.
<p align='center'>
<img src="https://user-images.githubusercontent.com/18648835/135598503-e54151a4-e127-4443-b82f-98a86cb5a33c.png" width="30%" />
</p>

Reply:
Using the placeholder from `articles.tsx` will solve the problem. It uses `useNumColumns()` to calculate the correct number of columns.

Reply:
@olerichter00 This placeholder is what is being used in Articles.tsx. It will also not look good on tablets.  I don't think useNumColumns will work as it is a hook.

Reply:
I just checked and it calculates the correct number of columns on tablets. The placeholder is only a component so hooks should work as expected :)

Reply:
Ohhh. @olerichter00 I get you now. You are referring to Articles.tsx in the current master. I couldn't find it in this branch, I can now see this PR removes it. 
Regarding if he can use hooks or not, I was referring to the fact that he cannot use hooks if he actually wants to use the new Placeholder in ArtistArticlesQueryRenderer's renderPlaceholder as is. i.e `renderPlaceholder: ArticlesPlaceholder`. I was assuming there are other places in the app that uses it that way that he doesn't want to touch hence the need to use a non hooks approach in order to continue with what he already has. Thanks @olerichter00 
In that case, @rajsam003, I will suggest either of the following:
1. Use the already existing ArticlesPlaceholder in Articles.tsx.
2. Or if you still want to keep things as they currently are in this branch, then you'd have to do the following:
`in ArticlesList.tsx`
```suggestion
export const ArticlesPlaceholder = () => {
  const numColumns = useNumColumns()
```

`in render prop of ArtistArticlesQueryRenderer inside ArtistArticles.tsx`

```
render={renderWithPlaceholder({
        Container: ArtistArticlesContainer,
        renderPlaceholder: () => <ArticlesPlaceholder />, // instead of renderPlaceholder: ArticlesPlaceholder
      })}
```